2025-04-22 19:24:03 --- Archivo 2025-04-22 19.24.log creado con éxito en H:\Python\Administradores de Bases de Datos P. Final\static\logs\2025-04-22 19.24.log
2025-04-22 19:24:03 ---  * Debugger is active!
2025-04-22 19:24:03 ---  * Debugger PIN: 827-411-152
2025-04-22 19:24:08 --- 127.0.0.1 - - [22/Apr/2025 19:24:08] "[35m[1mGET / HTTP/1.1[0m" 500 -
2025-04-22 19:24:09 --- 127.0.0.1 - - [22/Apr/2025 19:24:09] "[36mGET /?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1[0m" 304 -
2025-04-22 19:24:09 --- 127.0.0.1 - - [22/Apr/2025 19:24:09] "[36mGET /?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1[0m" 304 -
2025-04-22 19:24:09 --- 127.0.0.1 - - [22/Apr/2025 19:24:09] "[36mGET /?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 304 -
2025-04-22 19:24:09 --- 127.0.0.1 - - [22/Apr/2025 19:24:09] "[36mGET /?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 304 -
2025-04-22 19:25:10 ---  * Detected change in 'H:\\Python\\Administradores de Bases de Datos P. Final\\app.py', reloading
2025-04-22 19:25:10 --- Servidor desconectado
